app/brave_main_delegate.cc

```diff
@@ -173,7 +173,6 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       features::kWebXr.name,
       features::kWebXrGamepadModule.name,
       unified_consent::kUnifiedConsent.name,
-      switches::kSyncUSSBookmarks.name,
```

Comment:
header in this file as well? https://github.com/brave/brave-core/pull/4324/files#diff-931f170b705cce8579670910b176bdf2L16

Reply:
it is required for `switches::kSyncServiceURL`